src/test/java/broski/BroskiTest.java

```diff
@@ -0,0 +1,24 @@
+package broski;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import broski.output.Broski;
+
+public class BroskiTest {
+    @Test
+    public void start_checkGreeting_success(){
```

Comment:
I like that the naming of the test methods are according to the Java coding standard.